documentation/list.yaml

```diff
@@ -0,0 +1,66 @@
+securityDefinitions:
+  APIKeyHeader:
+     type: apiKey
+     in: header
+     name: Authorization
+
+paths:
+  /list/:
+    post:
+      security:
+        - APIKeyHeader: []
+      tags:
+        - List
+      summary: Posts a list
+      produces:
+        - application/json
+      parameters:
+        - in: body
+          name: body
+          required: true
+          example:
+            isDefault: false
+            name: "List's name"
+            isPublic: true
+            mediaItems: [
+              {
+                type: "movie",
+                id: 111,
+                title: "A night at the mauseleum",
+                description: "the remake of a night at the museum, but low budget",
+                imagePath: "thismyurl.com/image.png",
+                genres: ["thriller", "comedie", "low-budget"],
+                year: 2020
+              },
+              {
+                type: "movie",
+                id: 222,
+                title: "pink floyd",
+                description: "that movie i cant remember the name where the hammers walk",
+                imagePath: "willthisgetanimage.com/image.png",
+                genres: ["Drama", "Philossophy", "Psychedelich"],
+                year: 1998
+              }
+            ]
+      responses:
+        200:
+          description: List was created succesfuly
+        401:
+          description: Unclear
```

Comment:
The description here could be something like: `Authorization token was not provided`